multiqc/modules/isoseq/isoseq.py

```diff
@@ -0,0 +1,231 @@
+                value_counts[size] += 1
+
+            for n_CCS, size in value_counts.items():
+                if n_CCS == 2:
+                    plot_data[s_name]["=2"] = size
+                elif 2 < n_CCS < 11:
+                    plot_data[s_name]["3-10"] += size
+                elif 10 < n_CCS < 101:
+                    plot_data[s_name]["11-100"] += size
+                elif n_CCS > 100:
+                    plot_data[s_name][">100"] += size
+
+        cats = {
+            "=2": {
+                "name": "2 CCS",
+            },
+            "3-10": {
+                "name": "3 to 10 CCS",
+            },
+            "11-100": {
+                "name": "11 to 100 CCS",
+            },
+            ">100": {
+                "name": "More than 100 CCS",
+            },
+        }
+
+        self.add_section(
+            name="Cluster size distribution",
+            anchor="isoseq-cluster-size-distribution",
+            description="A distribution of cluster size (number of CC clustered to form one Hifi read).",
+            helptext="""
+            The csv file produced by Iso-Seq cluster shows which CCS have been clustered together to form 
+            one Hifi reads. The bargraph represent the distribution of the cluster size using four 
+            categories : -2, 3-10, 11-100, >100.
+            """,
+            plot=bargraph.plot(
+                plot_data,
+                cats,
+                {
+                    "id": "isoseq-cluster-size-distribution-barplot",
+                    "title": "Iso-Seq cluster: Histogram of cluster size.",
+                    "ylab": "Count",  # Y axis label
+                },
+            ),
+        )
+
+    def _add_general_stats_refine(self, data_by_sample):
+        headers = {
+            "num_reads_fl": {
+                "title": "Full-length",
+                "description": "Number of CCS where both primers have been detected",
+                "scale": "GnBu",
+                "format": "{:,.d}",
+            },
+            "num_reads_flnc": {
+                "title": "Non-chimeric full-length",
+                "description": "Number of non-chimeric CCS where both primers have been detected",
+                "scale": "RdYlGn",
+                "format": "{:,.d}",
+            },
+            "num_reads_flnc_polya": {
+                "title": "Poly(A) free non-chimeric full-length",
+                "description": "Number of non-chimeric CCS where both primers have been detected and the poly(A) tail has been removed",
+                "scale": "GnBu",
+                "format": "{:,.d}",
+            },
+        }
+        self.general_stats_addcols(data_by_sample, headers, namespace="refine")
+
+    def _add_refine_plot(self, values_by_metric_by_sample):
+        data_labels = {
+            "fivelen": "5' primer length",
+            "threelen": "3' primer length",
+            "insertlen": "Insert length",
+            "polyAlen": "Poly(A) length",
```

Comment:
I think that these two can be their own separate plots, rather than tabbed data labels.